migration/sql-files/073-labels.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE labels (
+    created_at timestamp with time zone,
+    updated_at timestamp with time zone,
+    deleted_at timestamp with time zone,
+    id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+    name text NOT NULL CHECK(name <> ''),
+    color text NOT NULL,
```

Comment:
I think it makes sense to have a default value of `#FFFFFF` for color.

Does it make sense to have a domain for this color field? For example:
```sql
CREATE DOMAIN hex_color AS TEXT
CHECK(
   VALUE ~ '^#\d{6}$'
);
```

@aslakknutsen what do you think?

Reply:
I also think we should have a distinct font color stored here. According to today's cabal.

Reply:
I have added the constraint directly to text and back ground colors.